# Designed to work with the run_batch_sim.py scrip. 
# List of experiments that will be iterated by the scrip. 
# Each experiment has a set of parameters that can be updated.
experiments:
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "norate"
    onnx_path: "rsl_rl/kingfisher_direct/noratelimit-seed42/exported/policy.onnx"
    localization_delay: 0
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "norate-ld100"
    onnx_path: "rsl_rl/kingfisher_direct/noratelimit-seed42/exported/policy.onnx"
    localization_delay: 100
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "ideal"
    onnx_path: "rsl_rl/kingfisher_direct/final-seed42/exported/policy.onnx"
    localization_delay: 0
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "ld100"
    onnx_path: "rsl_rl/kingfisher_direct/final-seed42/exported/policy.onnx"
    localization_delay: 100
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "loe90"
    onnx_path: "rsl_rl/kingfisher_direct/final-seed42/exported/policy.onnx"
    loe_right: 0.9
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "dist"
    onnx_path: "rsl_rl/kingfisher_direct/final-seed42/exported/policy.onnx"
    mass: 50.0
    cog_y: 0.5
    sdq: 25.89
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "pn25"
    onnx_path: "rsl_rl/kingfisher_direct/final-seed42/exported/policy.onnx"
    pixel_noise_radius: 25
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "all-light"
    onnx_path: "rsl_rl/kingfisher_direct/final-seed42/exported/policy.onnx"
    loe_right: 0.9
    pixel_noise_radius: 25
    mass: 50.0
    cog_y: 0.5
    sdq: 25.89
  - base_dir: "/home/luis/workspaces/bags_ws/tfr_exp/sim_bags"
    exp_name: "all-heavy"
    onnx_path: "rsl_rl/kingfisher_direct/final-seed42/exported/policy.onnx"
    localization_delay: 0
    loe_right: 0.6
    pixel_noise_radius: 50
    mass: 50.0
    cog_y: 0.1
    sdq: 34.52  
  